src/App.tsx

```diff
@@ -15,36 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  None = '',
+  Alphabetically = 'alphabetically',
+  Length = 'length',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.None);
```

Comment:
You can remove `<SortType>` because **TypeScript** figures out the **type** from `SortType.None` automatically. This makes the code cleaner and easier to read. No need to write it twice!

```suggestion
export const App: React.FC = () => {
  const [sortField, setSortField] = useState(SortType.None);
```